### PR TITLE
An approach to implementing CI

### PR DESCRIPTION
**Pull request by vargas-jevd**
This is  an initial approach to implementing CI on Jedis project.
In order to achieve this goal, the project has been linked to Travis-CI.
Since Jedis is a JavaScript project, I'm using PhantomJS to orchestrate headless tests.
Basically, what I'm doing is running tests in lib/jedis-test.js with the help of PhantomJS.
The work is still in progress.
Some rework might have to be done on jedis-test.js in order to make CI possible under this approach.
I'm open to any feedback, from a quick pair of eyes on the code, to critique on design and technical approach.

### CODE REVIEW

**Review by lomefin — CHANGES_REQUESTED** [review]
As you can see in the rest of the files, indentation is used with 2 spaces instead of 4.

File: testrunner.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				case 'SETS':
+					jedisTest.testGroupSets();
+					break;
+				case 'LOCALSTORAGE':
+					jedisTest.testGroupLocalStorage();
+					break;	
+			}
+		}
+		catch(err) {
+			phantom.exit(-1);
```
Comment (lomefin) [inline testrunner.js]: If there is an error, maybe it should be printed out.

**Review by lomefin — CHANGES_REQUESTED** [review]

**Comment by vargas-jevd** [comment]
Changes requested has been made.
A new version of jedis-test.js was created.
With this new version is now possible to truly test Jedis using Travis-CI.

**Review by lomefin — APPROVED** [review]
Looks good.

**Pull request merged**